src/core/layertree/qgslayertreelayer.cpp

```diff
@@ -97,16 +97,17 @@ void QgsLayerTreeLayer::setName( const QString &n )
   }
 }
 
-QgsLayerTreeLayer *QgsLayerTreeLayer::readXml( QDomElement &element )
+QgsLayerTreeLayer *QgsLayerTreeLayer::readXml( QDomElement &element, const QgsReadWriteContext &context )
 {
+  Q_UNUSED( context );
```

Comment:
Not needed here?

Reply:
I think it is a leftover of my first iteration. I'll remove it, thanks for reviewing!